src/components/settings/owner/ReplaceOwnerDialog/index.tsx

```diff
@@ -38,8 +38,15 @@ export const ReplaceOwnerDialog = ({ address }: { address: string }) => {
     <div>
       <Track {...SETTINGS_EVENTS.SETUP.REPLACE_OWNER}>
         <Tooltip title="Replace owner">
-          <IconButton onClick={() => setOpen(true)}>
-            <ChangeCircleOutlinedIcon />
+          <IconButton
+            onClick={() => setOpen(true)}
+            sx={{
+              '& svg path': {
+                fill: ({ palette }) => palette.primary.main,
+              },
+            }}
```

Comment:
It seems it already has the correct color even without the override

Reply:
I'd sooner keep this so that it matches the new palette when we implement it.